app/controllers/hyrax/file_sets_controller.rb

```diff
@@ -8,7 +8,7 @@ class FileSetsController < ApplicationController
     include Hyrax::Breadcrumbs
 
     before_action :authenticate_user!, except: [:show, :citation, :stats]
-    load_and_authorize_resource class: ::FileSet, except: :show
+    load_and_authorize_resource class: Hyrax.config.disable_wings ? ::Hyrax::FileSet : ::FileSet, except: :show
```

Comment:
Wondering if we want a `Hyrax.config.file_set_class` much like what we have for admin_set?  We could then encapsulate that logic there.

Reply:
This might be the way to go, although it could make it less obvious if a piece of code is designed for use only with legacy ::FileSet objects.